lib/containers/CreateNew.js

```diff
@@ -21,20 +34,56 @@ class CreateNew extends PureComponent {
     return {};
   }
 
+  async select(key, value) {
+    await this.setState(
+      Object.assign({}, this.state, {
+        [key]: value,
+      })
+    );
+  }
+
   render() {
     const { createWallet } = this.props;
-    const tabs = [
-      { header: 'New Wallet', body: <NewWallet createWallet={createWallet} /> },
-      { header: 'Import Wallet', body: <ImportWallet /> },
-      {
-        header: 'NewMultisigWallet',
-        body: <NewMultisigWallet createWallet={createWallet} />,
-      },
-    ];
+    const { keyManagement, walletType, walletOpt } = this.state;
+
+    let render;
+
+    /*
+     * calculate what to render to attempt to make easy to read
+     * the definitions for these values are found in PreselectOptions
+     */
+    const renderNewWallet =
+      walletType === 'personal' &&
+      keyManagement === 'node' &&
+      walletOpt === 'standard';
+
+    const renderHardwareImport =
+      walletType === 'personal' && keyManagement === 'hardware';
+
+    const renderWatchOnly =
+      walletType === 'personal' &&
+      keyManagement === 'node' &&
+      walletOpt === 'watch-only';
+
+    if (renderNewWallet) render = <NewWallet createWallet={createWallet} />;
+    else if (renderHardwareImport)
+      render = <ImportWallet createWallet={createWallet} />;
+    else if (renderWatchOnly)
+      render = <WatchOnly createWallet={createWallet} />;
+    else if (walletType === 'multiparty')
+      render = <NewMultisigWallet createWallet={createWallet} />;
+    else render = null;
+
     return (
       <div>
-        <Header type="h2">Create New</Header>
-        <TabMenu tabs={tabs} />
+        <PreselectOptions
+          keyManagement={keyManagement}
+          walletType={walletType}
+          select={this.select}
```

Comment:
I think we ran into this somewhere else maybe in simple wallet, but rather than bind you should just be able to use arrow functions, since they implicitly bind to the context, `select={ () => this.select() }` should work. `name-manager` uses this convention a lot since I didn't use any redux there and had to pass functions around to child components.

Reply:
Updated